Makefile

```diff
@@ -133,4 +141,10 @@ get-db:
 	docker pull ${OCT_IMAGE}
 	docker run -v ${REPO_DIR}/cmd/tnf/fetch:/tmp/dump:Z --user $(shell id -u):$(shell id -g) --env OCT_DUMP_ONLY=true ${OCT_IMAGE}
 delete-db:
-	rm -rf ${REPO_DIR}/cmd/tnf/fetch
\ No newline at end of file
+	rm -rf ${REPO_DIR}/cmd/tnf/fetch
+
+build-image-local: install-tools build-cnf-tests
```

Comment:
Isn't it more convenient to perform these steps inside the `Dockerfile`? Otherwise the validity of the image generated using this `Dockerfile` is dependent of being run through the `Makefile`.

Reply:
I'm 50/50.  On one hand it keeps it all contained within the Dockerfile, but makes it more complex because the environment already has all of the deps needed.

Reply:
I prefer leaving the `Dockerfile` as independent as possible, containing all the dependencies the building environment requires but would like to hear a second opinion here.

Reply:
I'll bring back the binary building within the Dockefile to encapsulate it.

Reply:
Done.